aminator/plugins/provisioner/apt.py

```diff
@@ -145,7 +145,7 @@ def apt_get_clean():
 
     @staticmethod
     def apt_get_install(*options):
-        install_result = monitor_command(['apt-get', '-y', 'install'] + options)
+        install_result = monitor_command(['apt-get', '-y', 'install'] + list(options))
```

Comment:
```
cmd = ['apt-get', '-y', 'install]
cmd.extend(options)
install_result = monitor_command(cmd)
```

Going that route looks a little cleaner and avoids the cast. Make the fix and I'll merge this.